.circleci/config.yml

```diff
@@ -843,15 +844,6 @@ workflows:
     jobs:
       - deploy-stored-build-job:
           study_key: << pipeline.parameters.study_key >>
-      - playwright-build:
-          env: << pipeline.parameters.deploy_env >>
-      - playwright-test:
-          env: << pipeline.parameters.deploy_env >>
-          parallelism_num: 2
-          test_suite: UNKNOWN
-          requires:
-            - playwright-build
-            - deploy-stored-build-job
 
```

Comment:
Removing this because `deploy-single-app` workflows are started individually for each study one after another on RC branch. Running all Playwright tests in every workflow at same time are major root cause of error 500 from back-end.